sig-release/README.md

```diff
@@ -0,0 +1,122 @@
+# SIG Release
+
+### Table of Contents
+- [Purpose](#purpose)
+- [Join us!](#join-us)
+- [Broad responsibilities](#broad-responsibilities)
+- [Specific responsibilities](#specific-responsibilities)
+- [Relation to the Release Manager and Team](#relation-to-the-release-manager-and-team)
+- [Ongoing responsibilities of the Release Team](#ongoing-responsibilities-of-the-release-team)
+  - ["Minor" Releases](#minor-releases)
+  - [“Patch” releases](#patch-releases)
+  - ["Security" releases](#security-releases)
+- [References](#references)
+
+## Join us!
+- [Google group](https://groups.google.com/forum/#!forum/kubernetes-sig-release)
+- [Slack channel](https://kubernetes.slack.com/messages/C56HWQ0TH/)
+- [Events and meetings calendar](https://calendar.google.com/calendar/embed?src=coreos.com_regcvcrgvq98lua2ikijg1g1uk%40group.calendar.google.com&ctz=America/Los_Angeles)
+
+## Purpose
+- Production of high quality Kubernetes releases on a reliable schedule
+- Ensure there is a [consistent group of community members](https://docs.google.com/document/d/1JAUqKl-lYdYLQ7GUT_9LzqxwQv-PcOdyAxNRZKItajo/edit) [1] in place to support the release process
+- Provide guidance and tooling to facilitate the production of automated releases
+- Serve as a tightly integrated partner with other SIGs to integration their repositories into the release process
+
+## Broad responsibilities
+- Ensuring high quality Kubernetes releases
+  - Define and staff release roles to manage resolving and communicating the status of release blocking criteria
+  - Define and drive development processes (e.g. merge queues, cherrypicks) and release processes 
+    (e.g. burndown meetings, cutting beta releases) with the intent of meeting the release timeline and delivering an 
+    on time release
```

Comment:
also, I think release/burndown process has been different for each release.  I think this group is responsible for standardizing that (e.g. all issues in the milestone block the release, using blocking/non-blocking labels, test flake count, etc).